### PR TITLE
Add north arrows and scale bars to map figures

### PR DESCRIPTION
**Pull request by rosypctals**

### CODE REVIEW

**Comment by rosypctals** [comment]
I've added the text from test.txt after running pip list >test.txt in the issue and updated the requirements.txt! I'll also update the documentation

**Comment by lkoolik** [comment]
Hi Amy, 

This looks great! No notes on your current edits. 

I tested the code with different configurations of the run options and realized we need to make the same edit in the `visualize_pwm_conc` function in `environmental_justice_calcs.py`.  Can you please make that edit, too? Then I can take a look and we can merge!

**Comment by rosypctals** [comment]
I've added the same code to the visualize_pwm_conc function! I understand that this function is called when the user inputs an output resolution larger than the ISRM grid- what would an example of that be?

**Comment by lkoolik** [comment]
> I've added the same code to the visualize_pwm_conc function! I understand that this function is called when the user inputs an output resolution larger than the ISRM grid- what would an example of that be?

Try running a test where the control file has the following option:
- OUTPUT RESOLUTION: C

File: requirements.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ fonttools==4.32.0
 geopandas==0.10.2
 kiwisolver==1.4.2
 matplotlib==3.5.1
+matplotlib-scalebar==0.8.1
```
[the remark this answers is not in the document]
Comment (lkoolik) [reply requirements.txt:13]: whoops! you already did this -- great job!

**Review by lkoolik — CHANGES_REQUESTED** [review]
I think I need to click this "Finish your review" in order for it to show you these comments? I'm not sure but I am clicking it anyway

File: scripts/tool_utils.py [hunk 2]
```diff
@@ -309,4 +312,58 @@ def get_output_region(region_of_interest, region_category, output_geometry_fps,
...
         
-    return output_region
\ No newline at end of file
+    return output_region
+
+def calculate_true_north_angle(center_lon, center_lat, crs):
+    """
+        Calculate the angle between the positive y-axis and true north.
+        
+        Parameters:
```
[the remark this answers is not in the document]
Comment (rosypctals) [reply scripts/tool_utils.py]: I've reformatted the two methods to follow this!

File: supporting/concentration_layer.py [hunk 3]
```diff
@@ -231,6 +269,93 @@ def process_emissions(self, emis, isrm_obj, verbose, output_dir, output_emis_fla
             
         return tmp_dct['PM25'], tmp_dct['NH3'], tmp_dct['VOC'], tmp_dct['NOX'], tmp_dct['SOX']
     
+    def visualize_individual_emissions(self, pollutant_name=''):
```
[the remark this answers is not in the document]
Comment (rosypctals) [reply supporting/concentration_layer.py]: Yes, I'm so sorry! I removed this method and fixed the error in run_echo_air.py- I'll make sure to keep the code more separate in the future!

**Review by lkoolik — APPROVED** [review]

**Pull request merged**